packages/bpk-component-page-indicator/src/BpkPageIndicator.js

```diff
@@ -36,19 +36,19 @@ export const VARIANT = {
 };
 
 export type Props = {
-  className: ?string,
```

Comment:
I was hesitant to get rid of it because a recent rule is not to recommend `className`.
Based on your comment, I am confident to revert the change, thank you.